templates/dashboard/contact_detail.html

```diff
@@ -70,4 +72,37 @@ <h1 class="h3">
   </div>
 {% endblock %}
 
+{% block script %}
+    {% if current_user.is_premium() %}
```

Comment:
The whole PGP section is disabled if user isn't premium so I think we don't have to do this check here.

Reply:
Removed